style.css

```diff
@@ -9,16 +9,27 @@
 */
 
 body {
-	margin: 80px;
-	background: #fff;
+   margin: 40px;
 	font-family: 'Poiret One', cursive;
    font-size: 20px;
-   background-color: #F5FFFA;
+   background-color: #F0F8FF;
+}
+
+img {
+     background-image: url("ghettomunda.jpg");
+     background-size: cover;
+     display: inline-block;
+}
+
+div.heading {
+   padding: 0 20px 20px 40px;
+   background-color: #ADD8E6;
+   margin-bottom: 50px;
 }
 
 h1 {
    float: left;
-	color: #00E5EE;
+	color: #1E90FF;
```

Comment:
Also indentation, spend some time figuring out how to set up Sublime to handle this correctly. I promise future Lelethu will thank you.